streamwebs_frontend/streamwebs/templates/streamwebs/index.html

```diff
@@ -1,21 +1,23 @@
+{% extends "streamwebs/base.html" %}
+{% load i18n %}
+
 <!DOCTYPE html>
 <html>
     <head>
         <title>StreamWebs</title>
     </head>
 
     <body>
```

Comment:
Since the base template already has the html structure, this doesn't need the doctype, html, head or body tags - just the content inside the body tags.